index.js

```diff
@@ -53,10 +53,10 @@ export const loadBlob = async (url, options = {}) =>
  * Load an item, parse the Response as blob and create a HTML Image.
  * @function
  * @param {string | import("./types.js").ImageOptions} urlOrOpts
- * @param {RequestInit} options
+ * @param {RequestInit} [options={}]
  * @returns {Promise<HTMLImageElement>}
  */
-export const loadImage = async (urlOrOpts, options = {}) => {
+export const loadImage = async (urlOrOpts, options) => {
```

Comment:
What's the use case of `Opts` in `urlOrOpts` ? Why would I have `loadImage({url})` vs `loadImage(url)` as for loadBlob to trigger i need to provide `options` anyway?